cypress/support/commands.ts

```diff
@@ -54,7 +54,9 @@ Cypress.Commands.add("deleteMachine", (hostname: string) => {
   cy.visit(generateMAASURL("/machines"));
   cy.findByRole("combobox", { name: "Group by" }).select("No grouping");
   cy.findByRole("searchbox").type(hostname);
-  cy.findByText(/Showing 1 out of 1 machines/).should("exist");
+  cy.findByText(/Showing 1 out of 1 machines/, { timeout: 15000 }).should(
```

Comment:
```suggestion
  cy.findByText(/Showing 1 out of 1 machines/, { timeout: LONG_TIMEOUT }).should(
```

Reply:
We have a `LONG_TIMEOUT` constant in `cypress/constants.ts` which would be suitable here.